lang/test/org/partiql/lang/eval/EvaluationSessionTest.kt

```diff
@@ -52,8 +57,21 @@ class EvaluationSessionTest {
     fun settingGlobals() {
         val globals = Bindings.empty<ExprValue>()
         val block: () -> EvaluationSession = { EvaluationSession.build { globals(globals) }}
+        val session = block.invoke()
 
-        assertEquals(globals, block.invoke().globals)
+        assertEquals(globals, session.globals)
+        assertEquals(ZoneOffset.UTC, session.defaultTimezoneOffset)
+        assertNow(block)
+    }
+
+    @Test
+    fun settingDefaultTimezoneOffset() {
+        val defaultTimezoneOffset = ZoneOffset.UTC
```

Comment:
For this test, I would put a different `ZoneOffset` other than the default of UTC to check that it's actually getting set.